### DISCUSSION TITLE
[Bug]: "Can I currently attend this meeting in person?" radio button does not save

### DISCUSSION THREAD

**Discussion by julien-paris** [code4recovery/12-step-meeting-list#663] [Q&A] [answered]
### Contact Details

ecomm.cer@aamail.org

### Website With Issue

https://alcoholics-anonymous.eu/

### What happened?

Please see the video
For certain Locations, I am now unable to save the meetings as "In person". 
This only happens for some locations. 
Almost all of the plugins were deactivated for debugging - no help there. 
Please advise. 
https://www.loom.com/share/c65c072d942e4b4fa29ed66f833456bb

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

### What browsers are you seeing the problem on?

Chrome

**Comment by tech2serve** [comment]
@julien-paris,

Thank you for the screen cast. Those are helpful.

We’ve seen this behavior a couple of times, and it has shown to be an issue with geocoding or cache. Please open a dev console and watch the output, and you should see a variable labeled `Geocoded`. Please pick an address that doesn’t work, and screen cap/attach the details of that variable. In particular, we want to inspect the value of `approximate`.

I entered “Engelburgergasse 11, 93047 Regensburg, GE” as a new meeting in my dev site, and it first geocoded to “Engelburgergasse 2, 93047…” but with a `formatted_address` of “Engelburgergasse, 93047…” which is probably what was cached. However, this still doesn’t evaluate on my site as approximate, meaning the meeting should save as “in-person”. However, if on your site, one of these addresses is indeed showing `approximate` as `yes`, we need to figure out why.

Please note: For our work flow, we’re asking that user issues be opened inside of Discussions first. The maintainers will then migrate the details to an Issue when appropriate. Thanks!

**Comment by julien-paris** [comment]
Thanks for the answer. 
This one does not work for example, bbut says "Approximate: no"
![image](https://user-images.githubusercontent.com/73740354/151842525-06f31a5b-ca67-4414-803b-3c0933ab4a5d.png)

**Reply by tech2serve** [reply]
And I’m able to publish that as well as an in-person meeting.

Are you seeing any unexpected errors in the console when you try and publish/update?

Are you specifically trying to migrate a meeting from online-only to in-person? Was this meeting previously published? If it was online, did you delete the online meeting details as part of your update attempt? If so, I’m wondering if making it hybrid first would work?

If none of this reveals the cause of the issue, then please spell out of the details of your setup. In particular, please confirm you’re using Wordpress 5.9. What PHP, etc? Have you tried other browsers?

**Reply by julien-paris** [reply]
No errors found, that was the full console. 
This meeting was published a long time ago, and I think went online then back in person. However, that is the case for quite a few of the meetings. 
We've tried other browsers because the issue is present for me as for the rest of the update team.
I am on Wordpress 5.9 on the staging site (the issue arises both on staging and prod)
I tried making it hybrid but same issue: 
https://www.loom.com/share/7e7727e3cff44773960c539da22739e9

**Reply by julien-paris** [reply]
Hi, am following up on this issue, which remains unresolved. 
I have update the production site and we are now on Wordpress 5.9, php 7.4. The theme is Hello Elementor. 
I have switched from Mapbox to Google maps API.
To no avail.
Any help would be appreciated, 
Thanks

**Comment by kiyote33** [comment]
Hi Julien,
I've loaded my sandbox with the **Hello Elementor** theme and have run a series of tests using my local meeting information as a data source.  I'm still not able to replicate your issue, as I get the expected result whenever I **reset an online meeting to in-person** with a valid street address. Could you provide a link to the data source which contains an example of this issue? If I can successfully replicate your issue in my sandbox, I have a better chance at suggesting a possible solution. Thanks for your help and patience on this...

**Reply by julien-paris** [reply]
Thank to you for your help. 
Is the data source the link? The json file? 
Also, would you like admin access to take a look? 

This is one example:
https://alcoholics-anonymous.eu/meetings/munich-aa-friday/

**Reply by kiyote33** [reply]
Yes, the data source json file link for Germany is what I need to get my configuration set the same as yours.  Thx...

**Reply by julien-paris** [reply]
I sent it by mail to devs@code4recovery.org
and it came back undelivered

**Comment by kiyote33** [comment]
Hi @julien-paris,
After using your data source link to load in meetings for Germany, I way able to edit the same meeting you use in your loom (Munich AA Friday). As demonstated in your loom, the usual expected behavior does **not** occur when resetting the radio buttons from No to Yes, indicating that there is indeed an in-person meeting at the meeting location. I will need to dig into this tomorrow to try and get more of a handle on it.  If we can get a work-around, I will pass that on to you as soon as it is devised.  Otherwise we will transition this discussion into a bug issue to be addressed by our maintenance team.

**Reply by julien-paris** [reply]
Thanks @kiyote33 I'll stand by

**Reply by kiyote33** [reply]
Having difficulty recreating the issue. I thought I had it late Monday, but unable to replicate it now.  The only thing I found that would cause your described issue for the inactive **Munich AA Friday** meeting is the approximate field is marked 'yes' on import. Normally an approximate address is marked 'yes' for an online-only meeting only. But cycling through the different location/address options seems to reset it correctly (i.e. approximate = 'no'.  Can you identify other meetings that might be having the same issue:

**Reply by julien-paris** [reply]
Sure here are some other examples: 

https://alcoholics-anonymous.eu/meetings/antwerp-international-aa-group/
https://alcoholics-anonymous.eu/meetings/friday-night-linz-step-meeting/

So these, like Munich, are marked as inactive.

**Reply by kiyote33** [reply]
Could you attach your CSV file in this window, so I can take a look at it?

**Reply by julien-paris** [reply]
[meetings.csv](https://github.com/code4recovery/12-step-meeting-list/files/8036432/meetings.csv)

**Reply by kiyote33** [reply]
thx...

**Comment by kiyote33** [comment, marked as answer]
Hi @julien-paris,
@joshreisner jumped in to help provide a solution here. He's opened a maintenance issue (**Location Approximate Not Updating #677**) to get a fix put in place for what we believe was the culprit for you not being able to update inactive meetings. Will let you know as soon as a patch is in place.

**Reply by julien-paris** [reply]
great news, thanks or the followup
